.github/workflows/ci.yml

```diff
@@ -13,12 +13,17 @@ defaults:
 jobs:
   test:
     name: Test C# ${{ matrix.dotnet }}
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
`ubuntu-20.04` was removed on April 15, so updated here, but also had to install libssl1.1 because we're targeting .NET 3.1. We should consider a major bump soon to drop .NET 3.1 support.